### DISCUSSION TITLE
Resize to top-left in absolute mode

### DISCUSSION THREAD

**Discussion by badiuciprian** [GrapesJS/grapesjs#4975] [Q&A]
Hi guys, I'm using absolute mode and it works great, there is just one 'glitch' :) when resizing the components from top or left handles: the component resizes on the opposite side, the top-left corner always remain the same. I think is the way the editor works coming from the time when absolut mode did not exist I assume, but is there a way to work this out? Thanks.

**Comment by PaulRill00** [comment]
I have just asked the same thing and provided a solution here: https://github.com/GrapesJS/grapesjs/discussions/5015

**Reply by artf** [reply]
Merged, thanks @PaulRill00

**Reply by badiuciprian** [reply]
cool, this means that it will be available on the next update?

**Reply by artf** [reply]
Available here: https://github.com/GrapesJS/grapesjs/releases/tag/v0.21.1

**Comment by badiuciprian** [comment]
cool, tanks!
